server/datastore/mysql/mysql.go

```diff
@@ -94,6 +94,20 @@ func retryableError(err error) bool {
 	return false
 }
 
+// TODO: The Adhoc family of functions allow packages outside mysql to run
```

Comment:
Turned out to use just one, at first I had `AdhocReader`, `AdhocWriter` and `AdhocTx` too.